### PR TITLE
Validate Pools

### PR DESCRIPTION
**Pull request by Satisha10**
Validation of `EastPools`, `UpperPools` and `WestPools` for Kii difficulty.

There is also a change in Gorlek for `EastPools.FightRoomHC`, and a new path in Moki for `WestPools.BurrowOre` (since there was no path in Moki for this pickup).

### CODE REVIEW

File: wotw_seedgen/areas.wotw [hunk 1]
```diff
@@ -11295,26 +11301,29 @@ anchor EastPools.LupoArea at -1409, -4163:  # Checkpoint to the left of Lupo
...
   pickup EastPools.FightRoomHC:  # The HC only gets unlocked if you finish the alternate WaterDash arena; FightArenas doesn't have WaterDash as requirement included for reasons explained above, so it's needed here
     moki, EastPools.FightArenas, WaterDash:
       DoubleJump, Dash OR Grapple
       Bash, Grenade=1
       Launch
-    gorlek, EastPools.FightArenas, WaterDash: Water OR Damage=20  # WaterDash on the bubble
-    kii, EastPools.FightArenas, WaterDash: DoubleJump  #@validator that might be gorlek? gorlek for sure with tjump
+    gorlek, EastPools.FightArenas, WaterDash:
+      Water OR Damage=20  # WaterDash on the bubble
+      DoubleJump
```
Comment (SiriusAshling) [inline wotw_seedgen/areas.wotw:11326]: I think gorlek would be better with TripleJump. It only seems to work leniently when jumping from the left side and using higher ground as starting point isn't intuitive to all players.

File: wotw_seedgen/areas.wotw [hunk 2]
```diff
@@ -12232,12 +12271,15 @@ anchor WestPools.Teleporter at -1656, -4171:  # the west Pools TP
       Sentry=2 OR Blaze=2
   pickup WestPools.BurrowEX:
     moki: Water, Burrow
-    kii: Damage=40, Burrow
-  pickup WestPools.BurrowOre:  # intended path omitted in moki
```
Comment (SiriusAshling) [inline wotw_seedgen/areas.wotw:12236]: This was intentional, some player can't collect this pickup even with all abilities available.
Logically unreachable pickups receive spirit light, so this is a good solution to ensure all moki players can complete their seeds.

**Review by SiriusAshling — CHANGES_REQUESTED** [review]

File: wotw_seedgen/areas.wotw (hunk 1 above)
Comment (Satisha10) [reply wotw_seedgen/areas.wotw]: I added triple jump in Gorlek, and moved double jump back to Kii.

File: wotw_seedgen/areas.wotw (hunk 2 above)
Comment (Satisha10) [reply wotw_seedgen/areas.wotw:12236]: Sorry about that, I did not interpret the comment correctly. I removed the path and explained in the comment why it is omitted.

**Review by SiriusAshling — APPROVED** [review]

**Comment by SiriusAshling** [comment]
Thanks! Merged manually to avoid including the changes on BeetleDefeated and in Woods. They aren't related and it would be confusing if they appeared in a commit that claims in its name to only change pools.

**Pull request closed without merging**